compiler/gentree.cpp

```diff
@@ -97,6 +99,9 @@ VertexPtr GenTree::get_foreach_value() {
 }
 
 VertexAdaptor<op_var> GenTree::get_function_use_var_name_ref() {
+  if (cur->type() == tok_clpar) { // for trailing comma
+    return {};
+  }
```

Comment:
Seems strange, because `get_func_param()` and other list-item parsing function don't handle ')' separately